docs/maya/muscle.md

```diff
@@ -212,14 +214,19 @@ To enable the debugger the *Debug* checkbox must be marked. To select the specif
   <figcaption><b>Figure 9</b>: AdnMuscle debug features. From left to right: Muscle Fibers, Attachment To Transform Constraints, Slide On Segment Constraints, Attachment To Geometry Constraints, Slide On Geometry Constraints and Shape Preservation.</figcaption>
 </figure>
 
+<figure markdown>
+  ![muscle fibers activation debug](images/muscle_fibers_activation_debug.png)
+  <figcaption><b>Figure 10</b>: On the left side the Muscle Fibers mode is set and the fibers are displayed when the muscle is not activated. On the right side the Muscle Fibers mode is set and the fibers are displayed when the muscle is not activated. The activated color has been changed to yellow. </figcaption>
```

Comment:
Is this correct? If so, same applies to ribbon.
```suggestion
  <figcaption><b>Figure 10</b>: On the left side the Muscle Fibers mode is set and the fibers are displayed when the muscle is not activated. On the right side the Muscle Fibers mode is set and the fibers are displayed when the muscle is activated. The activated color has been changed to yellow. </figcaption>
```

Reply:
You are correct. My bad, thanks!